README.md

```diff
@@ -123,6 +123,7 @@ Also check [Before you install](https://docs.bigbluebutton.org/2.3/install.html#
 | ⚠️ | `bbb_freeswitch_default_password` | set the default password for freeswitch | | Can be generated with `pwgen -s 16 1` |
 | | `bbb_html5_backend_processes` | amount of html5 backend processes | 1 | min = 1; max = 4 |
 | | `bbb_html5_frontend_processes` | amount of html5 frontend processes | 1 | min = 1; max = 4; or 0 to let the same process do front- and backend (2.2 behavior) |
+| | `bbb_html5_frontend_max_old_space_size` | max-old-space-size in frontends | `2048` | |
```

Comment:
a description of the usage would be good, like when to increase or decrease this value, what the consequences are